mix.exs

```diff
@@ -34,7 +34,7 @@ defmodule Phoenix.Mixfile do
 
   def application do
     [mod: {Phoenix, []},
-     applications: [:plug, :poison, :logger, :eex],
+     extra_applications: [:logger, :eex],
```

Comment:
Therefore, let's revert this change BUT leave a TODO so we update it as soon as 1.3 is out.